### PR TITLE
User story 27

### LINKED ISSUE #29: User Story 27, User Profile displays Orders link
As a registered user
When I visit my Profile page
And I have orders placed in the system
Then I see a link on my profile page called "My Orders"
When I click this link my URI path is "/profile/orders"

### PR DESCRIPTION
**Pull request by philjdelong**
## Description
Resolves #29 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Refactor (non-breaking change which maintains existing functionality)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## RSpec Results
```
Finished in 4.74 seconds (files took 2.41 seconds to load)
140 examples, 0 failures, 2 pending

Coverage report generated for RSpec to /Users/phildelong/turing/mod2/projects/monster_shop_part_1/cove
rage. 1744 / 1820 LOC (95.82%) covered.

**(This will work itself out when we refactor for order creation)**
```

### CODE REVIEW

File: spec/features/merchants/destroy_spec.rb [hunk 1]
```diff
@@ -25,7 +25,7 @@
       expect(page).to_not have_content("Brian's Bike Shop")
     end
 
-    it "I can't delete a merchant that has orders" do
+    xit "I can't delete a merchant that has orders" do
```
Comment (DanielEFrampton) [inline spec/features/merchants/destroy_spec.rb:28]: @mel-rob @KirbyDD Phil and I talked about this; this test will get fixed in my next PR.

File: spec/features/orders/creation_spec.rb [hunk 2]
```diff
@@ -28,7 +28,7 @@
       click_on "Checkout"
     end
 
-    it 'I can create a new order' do
+    xit 'I can create a new order' do
```
Comment (DanielEFrampton) [inline spec/features/orders/creation_spec.rb:31]: See above comment.

**Review by DanielEFrampton — APPROVED** [review]
Thanks for tackling this!

**Review by mel-rob — APPROVED** [review]
This looks great. Excited to see those tests pass!

**Pull request merged**